docs/sources/gard.md

```diff
@@ -0,0 +1,86 @@
+# MONDO - GARD Alignment
```

Comment:
Added: GARD docs
@matentzn I created this page and then added over the stuff I wrote in the [readme](https://docs.google.com/document/d/1sM0B0ICdOWDwZWt0M_Y3t5QYwqL-XwOsDl3TS14XRa4/edit#) about my analysis. While doing this, I noticed that the formatting in these `docs/sources/` page could be improved: (i) possibly using sub-headers instead of the **bolded:** style, (ii) the last section where it talks about running `make` is cookie cutter, whereas most of these sources don't have a specific comments section. I was thinking maybe the comments should go at the bottom. Or, it might be nice to split the comments from the `make` section using a horizontal rule, which I did in this document by adding `---`.

Reply:
Feel free to follow you guts here - I am not particularly wedded to anything!